core/server/models/base/index.js

```diff
@@ -717,20 +715,20 @@ ghostBookshelf.Model = ghostBookshelf.Model.extend({
         }
 
         return itemCollection.fetchPage(options).then(function formatResponse(response) {
-            var data = {},
-                models;
-
-            options.columns = requestedColumns;
-            models = response.collection.toJSON(options);
+            // Filtering attributes here so they are not leaked to api layer
+            // where models are serialized to json and do not do more filtering
```

Comment:
I find the old comment helpful too. It explains that we extend the original columns currently.

Reply:
[Added](https://github.com/TryGhost/Ghost/pull/9899/files#diff-f8d02ad12b13d44b912bbf46cf7193b6R668) back the previous comment with adjustment to how filtering being done now

Reply:
Sorry i meant

> // re-add any computed properties that were stripped out before the call to fetchPage
> // pick only requested before returning JSON